privacy-protections/storage-blocking/index.html

```diff
@@ -8,6 +8,7 @@
     <script src='./helpers/idb-wrapper.js' defer></script>
     <script src='./helpers/commonTests.js'></script>
     <script src='./main.js' defer></script>
+    <script src='https://good.third-party.site/privacy-protections/storage-blocking/3rdparty.js' defer></script>
```

Comment:
If you'll run this test page with `?store` or `?retrieve` main.js will run the tests immediately and your test might be injected too late to run. How about this:

```html
<script>const tests = []</script>
<script src="…3rdparty.js" defer></script>
<script src="…main.js" defer></script>
```

Reply:
Sure, I didn't know about that option, I'll make the change.